database/create_tables.sql

```diff
@@ -10,62 +23,62 @@ CREATE TABLE Projects (
     project_name varchar(255) PRIMARY KEY,
     project_description text,
     project_image_url varchar(255),
```

Comment:
Need to have the following details:
- project_name
- project_category
- project_image_url
- project_deadline
- project_funding_goal
- project_description

A snippet of dummy SQL I have created to include the above details

> create table projects (project_name varchar(255) PRIMARY KEY,
> project_category varchar(90),
> project_image_url varchar(255),
> project_deadline timestamp without time zone,
> project_funding_goal numeric,
> project_description text);`